openbb_terminal/cryptocurrency/overview/overview_controller.py

```diff
@@ -1572,6 +1573,7 @@ def call_news(self, other_args):
         )
 
         parser.add_argument(
+            "-f",
```

Comment:
I think we leave "-f" as a reserved argument for file paths, so this is by design and not a bug

Reply:
@jmaslek should we change it to `-fil` or just don't have a singular `-` flag for filter for this command?

Reply:
Just dont have a single `-` command.

Fun fact, we dont have (or shouldnt have) any multi-letter single dash commands.  Since it would interpret `-fil` as `-f -i -l` which can be problematic

Reply:
@jmaslek updated!